en/includes/guides/authentication/mfa/add-x509-login.md

```diff
@@ -1,157 +1,191 @@
+        authorityInfoAccess = OCSP;URI: http://clients1.google.com/ocsp
         ```
     
-    6. Now, get the `pkcs12` out of `.crt` file using the command given below as it is been used to import certificates to the browser.
+4. Once it is done, sign the CSR using the root CA key generated in Step 1 above.
 
-        ``` shell
-        keytool -importkeystore -srckeystore localcrt.jks -destkeystore localhost.p12 -srcstoretype JKS -deststoretype PKCS12 -srcstorepass 
-        localpwd -deststorepass browserpwd -srcalias localcrt -destalias browserKey -srckeypass localpwd -destkeypass browserpwd -noprompt
-        ```
+    ```shell
+    openssl ca -batch -startdate 150813080000Z -enddate 250813090000Z -keyfile rootCA.key -cert rootCA.crt -policy policy_anything -config <custom_config_location> -notext -out localcrt.crt -infiles localcrt.csr
+    ```
+
+    !!! note
+        The `-config` flag is only needed if you are using a custom configuration file.
 
-        Make sure to use the same password you used when creating the keystore for the `srcstorepass` in the above step. Now you have 
-        the `localhost.p12` file that you can import into your browser as explained in the [import certificate](#import-certificate) section.
+    This creates a signed certificate called `localcrt.crt` that is valid for a specified period that is denoted by the `startdate` and `enddate`.
 
-    7. Next, create a new trust store and import the server certificate into the trust store using the following commands:
+5. The next step is to import the CA and signed certificate into the keystore using the following commands.
 
-        ``` shell
-        keytool -import -keystore cacerts.jks -storepass cacertspassword -alias rootCA -file rootCA.crt -noprompt
-        keytool -importcert -alias localcrt -file localcrt.crt -keystore cacerts.jks -storepass cacertspassword -noprompt
-        ```
+    ```shell
+    keytool -importcert -alias rootCA -file rootCA.crt -keystore localcrt.jks -storepass localpwd -noprompt
+
+    keytool -importcert -alias localcrt -file demoCA/newcerts/01.pem -keystore localcrt.jks -storepass localpwd -noprompt
+    ```
+
+6. Use the commands below to convert the `.crt` file into the PKCS12 format, which is used for importing certificates into browsers:
+
+    ``` shell
+    keytool -importkeystore -srckeystore localcrt.jks -destkeystore localhost.p12 -srcstoretype JKS -deststoretype PKCS12 -srcstorepass localpwd -deststorepass browserpwd -srcalias localcrt -destalias browserKey -srckeypass localpwd -destkeypass browserpwd -noprompt
+    ```
+
+    Make sure to use the same password you used when creating the keystore for the `srcstorepass` in the above step. Now you have the `localhost.p12` file that you can import into your browser as explained in the [import certificate](#import-certificate) section.
+
+7. Next, create a new trust store and import the server certificate into the trust store using the following commands:
+
+    ```shell
```

Comment:
Shall we also break this into two steps? It would make it easy to copy paste the command directly from the doc.
WDYT?